src/internet_identity/src/storage.rs

```diff
@@ -167,6 +246,22 @@ impl<T: candid::CandidType + serde::de::DeserializeOwned, M: Memory> Storage<T,
         self.flush();
     }
 
+    pub fn migration_state(&self) -> MigrationState {
+        match self.header.version {
+            1 => MigrationState::NotStarted,
+            2 if self.header.migration_batch_size == 0 => MigrationState::Paused {
+                anchors_left: self.header.new_layout_start as u64,
+                batch_size: self.header.migration_batch_size as u64,
+            },
```

Comment:
Do we even need a "Paused" state? Isn't this the same as having `migration_batch_size == 0`?

Reply:
Yes, it's the same. I just thought it was nicer to have it explicitly. `inProgress` without making further progress feels weird, so I would rename it to just `started` then...